docker-compose-cloud.yml

```diff
@@ -0,0 +1,166 @@
+version: "3.8"
```

Comment:
Do we really need a different docker-compose for the cloud deployment ?

Reply:
Unless we can cherry pick and merge stuff from two docker-compose, yes. I wasn't able to find a better way to do this but feel free to suggest something :eyes:

Reply:
This is hard to solve an otherway IMO because we need to "build" and "up" while still injecting the env variables for the

Reply:
Think more on it I guess I could simply make a second "cloud-web" entry that adds the env vars rather than. The makefile could the simply run everything with the normal web then when "up" is called it calls "cloud-web" instead of just "web"